oaknational/Oak-Web-Application
# Feat/lesq 561/specialist download page [LESQ-561]

## Description

Music year: 1974

- adds queries, schema and page for specialist lesson downloads
- next lesson links on downloads success page still TODO

## How to test

Yet another 'no changes in preview' PR. 
1. Go to https://deploy-preview-2306--oak-web-application.netlify.thenational.academy/teachers/specialist/programmes/independent-living-applying-learning/units/staying-safe-al-5556/lessons/online-safety-c5gk8r/downloads
2. Should be 'internal error' (404)
3. You should see the rest of the site remains the same

## Screenshots

How it should now look:

<img width="510" alt="Screenshot 2024-03-11 at 11 50 00" src="https://github.com/oaknational/Oak-Web-Application/assets/45038878/7eed9fce-7d2b-473c-9f16-048756d18c3d">
